### PR TITLE
Adds EZ-clean grenades to the uplink

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: Traitor janitors can now order EZ-clean grenades for 6 telecrystals per bundle. They function like normal cleaning grenades with an added "oh god my face is melting" effect, and can also be found in surplus crates.
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/grenades/chem_grenade.dm [hunk 1]
```diff
@@ -381,6 +381,25 @@
...
+	stage = READY
+
+/obj/item/weapon/grenade/chem_grenade/ez_clean/New()
+	..()
+	var/obj/item/weapon/reagent_containers/glass/beaker/large/B1 = new(src)
+	var/obj/item/weapon/reagent_containers/glass/beaker/large/B2 = new(src)
+
+	B1.reagents.add_reagent("fluorosurfactant", 40)
+	B2.reagents.add_reagent("water", 40)
+	B2.reagents.add_reagent("ez_clean", 60) //ensures a  t h i c c  distribution
```
Comment (coiax) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: Do you have to invent a new reagent?

Comment (Xhuis) [inline code/game/objects/items/weapons/grenades/chem_grenade.dm]: To keep the functionality that makes it look the exact same and still work as a cleaner, yes

**Comment by Armhulen** [comment]
these should clean a lot better than normal grenades. as in, they can clean bodies, floor tiles (careful), items, they should REALLY CLEAN.

**Comment by Xhuis** [comment]
They function as an organic-killer. A human that stands for the full duration in a puddle of foam from one of the grenades will take almost 80 total damage.

**Comment by Iamgoofball** [comment]
@Xhuis should outright kill entirely

**Comment by Armhulen** [comment]
just throw two @Iamgoofball

**Comment by Militaires** [comment]
this should clean people off the floor not just burn them

**Comment by Armhulen** [comment]
yeah that's kinda what I meant, leaving skeletons if it does over 200.

WHICH IS ANOTHER GREAT IDEA, LEAVE 200 IF YOU GO OVER 200 TOXINS

**Pull request merged**